clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -458,6 +482,10 @@ public boolean matches(AbstractRequest body) {
 
     private void awaitFirstHeartbeat(final AtomicBoolean heartbeatReceived) throws Exception {
         mockTime.sleep(HEARTBEAT_INTERVAL_MS);
+        // Awake the heartbeat thread
```

Comment:
I would prefer to remove it since it changes the behavior under test. Another way to speed up the test is perhaps to lower the default retry backoff. Maybe 20ms is good enough?